api/board/README.md

```diff
@@ -0,0 +1,150 @@
+Board API
+=========
+
+This API provides low level I/O operations supported by hardware boards.
+  - [GPIO - General Purpose I/O](./gpio.md)
+  - [AIO - Analog I/O](./aio.md)
+  - [PWM - Pulse Width Modulation](./pwm.md)
+  - [I2C - Inter-Integrated Circuit](./i2c.md)
+  - [SPI - Serial Peripheral Interface](./spi.md)
+  - [UART - Universal Asynchronous Receiver/Transmitter](./uart.md).
+
+This API uses board pin names as defined by implementations in the [`Board.pins()`](#getpins) and the [`Board.pin()`](#getpin) methods. Implementations SHOULD list pin names, pin addresses, supported modes, and other capabilities or constraints on how the pins can be configured.
+
+The full Web IDL definition for Board and IO APIs can be found [here](./webidl.md).
+
+The API object
+--------------
+The API entry point is a [`Board`](./#board) object that is exposed in a platform specific manner. As an example, on Node.js it can be obtained by requiring the package that implements this API. On other platforms, it can be constructed.
+
+```javascript
+var board = require('iot-board');
```

Comment:
Shouldn't you make it clear that this is probably not called iot-board? like '<iot-board-name>'